Example/src/screens/Animations.tsx

```diff
@@ -0,0 +1,180 @@
+import React, {useState, useLayoutEffect} from 'react';
+import {View, StyleSheet} from 'react-native';
+import {enableScreens} from 'react-native-screens';
+import {
+  createNativeStackNavigator,
+  NativeStackNavigationProp,
+} from 'react-native-screens/native-stack';
+import {SettingsPicker, Button} from '../shared';
+
+enableScreens();
+
+type StackParamList = {
+  Main: undefined;
+  Push: undefined;
+  Pop: undefined;
+  Screen: undefined;
+  Modal: undefined;
+};
+
+interface MainScreenProps {
+  navigation: NativeStackNavigationProp<StackParamList, 'Main'>;
+  stackAnimation: string;
+  setStackAnimation: (value: string) => void;
+}
+
+const MainScreen = ({
+  navigation,
+  stackAnimation,
+  setStackAnimation,
+}: MainScreenProps): JSX.Element => {
+  useLayoutEffect(() => {
+    navigation.setOptions({
+      stackAnimation,
+    });
+  }, [navigation, stackAnimation]);
+
+  return (
+    <View style={styles.container}>
+      <SettingsPicker
+        label="Stack animation"
+        value={stackAnimation}
+        onValueChange={setStackAnimation}
+        items={[
```

Comment:
Maybe it is possible to retrieve the keys of a `StackAnimationTypes`?

Reply:
Unfortunately, [you can't retrieve an array from a type](https://stackoverflow.com/a/59857409)